src/constants/themes/primaryTheme.ts

```diff
@@ -1,9 +1,11 @@
 import PRIMARY_COLORS from '../colors/primary';
 import PRIMARY_FONTS from '../fonts/primary';
+import { PRIMARY_TYPOGRAPHY } from '../typography/primaryTypography';
 
 export const primaryTheme = {
   __type: 'primary',
   BOX_SHADOW: {},
   COLORS: PRIMARY_COLORS,
   FONTS: PRIMARY_FONTS,
+  TYPOGRAPHY: PRIMARY_TYPOGRAPHY,
```

Comment:
Went with just `TYPOGRAPHY` instead of `TYPOGRAPHY_CONSTANTS` since we're accessing it off the `theme` now, so it is more obvious they are constants: there's no need to distinguish between `TYPOGRAPHY` and `TYPOGRAPHY_CONSTANTS` since they're not being imported from the same place like we currently do.